packages/wdio-sauce-service/src/service.ts

```diff
@@ -1,13 +1,78 @@
-import SauceLabs from 'saucelabs'
+import SauceLabs, { Job, SauceConnectOptions, SauceLabsOptions } from 'saucelabs'
 import logger from '@wdio/logger'
 import { isUnifiedPlatform } from './utils'
+import SuiteStats from '../../wdio-reporter/build/stats/suite'
```

Comment:
We can't access modules like that, it would fail when installed through NPM

```suggestion
import { SuiteStats } from '@wdio/reporter'
```